assets/js/hooks/ui-interactions.js

```
@@ -0,0 +1,279 @@
+      let targetCell = null;
+      
+      switch(e.key) {
+        case 'ArrowRight':
+          targetCell = focusedCell.nextElementSibling;
+          break;
+        case 'ArrowLeft':
+          targetCell = focusedCell.previousElementSibling;
+          break;
+        case 'ArrowDown':
+          // Move to same day next week (7 cells forward)
+          const nextRow = focusedCell.parentElement.nextElementSibling;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)